File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/ios/Classes/Model/IDOWatchDialIdModel.swift
//
//  IDOWatchDialIdModel.swift
//  protocol_channel
//
//  Created by hc on 2023/10/12.
//

import Foundation
/// Get watch ID event number
@objcMembers
public class IDOWatchDialIdModel: NSObject, IDOBaseModel {
    /// Watch ID
    public var watchId: Int

    enum CodingKeys: String, CodingKey {
        case watchId = "watch_id"
    }

    public init(watchId: Int) {
        self.watchId = watchId
    }

    public func toJsonString() -> String? {
        if let jsonData = try? JSONEncoder().encode(self) {
            let jsonString = String(data: jsonData, encoding: .utf8)
            return jsonString
        }
        return nil
    }
}

File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/android/src/main/kotlin/com/idosmart/model/IDOWatchDialIdModel.kt
//
//  IDOWatchDialIdModel.kt
//  protocol_channel
//
//  Created by hedongyang on 2023/10/30.
//

package com.idosmart.model

import com.google.gson.GsonBuilder
import com.google.gson.annotations.SerializedName

///
/// Get watch ID event number

open class IDOWatchDialIdModel(watchId: Int) : IDOBaseModel {

    /// Watch ID
    @SerializedName("watch_id")
    var watchId: Int = watchId


    

    override fun toJsonString(): String {
        return GsonBuilder().create().toJson(this).toString()
    }
}
    